doc/install.rst

```diff
@@ -94,35 +94,63 @@ Currently, ``xgboost-cpu`` package is provided for x86_64 (amd64) Linux and Wind
 Conda
 *****
 
-You may use the Conda packaging manager to install XGBoost:
+The `Conda <https://conda.io>`_ packaging manager provides a language agnostic, cross-platform friendly, ``sudo``-free install experience.
+
+If you don't already have Conda, it can be installed via `the conda-forge installer, Miniforge <https://conda-forge.org/download/>`_. 
+
+To install Python XGBoost:
 
 .. code-block:: bash
 
-   conda install -c conda-forge py-xgboost
+   conda install py-xgboost
 
-Conda should be able to detect the existence of a GPU on your machine and install the correct variant of XGBoost. If you run into issues, try indicating the variant explicitly:
+To install R XGBoost:
+
+.. code-block:: bash
+
+   conda install r-xgboost
+
+To install the XGBoost C++ library:
+
+.. code-block:: bash
+
+   conda install libxgboost
+
+Additionally XGBoost comes in CPU-only and CUDA variants.
+
+If you have an NVIDIA GPU with CUDA driver installed, Conda detects it and prefers the CUDA variant.
+
+If you would like to a specific variant, it can be selected explicitly.
+
+For the CPU variant it can be installed like so
 
 .. code-block:: bash
 
    # CPU only
-   conda install -c conda-forge py-xgboost-cpu
-   # Use NVIDIA GPU
-   conda install -c conda-forge py-xgboost-gpu
+   conda install -n xgboost 'libxgboost=*=cpu*'
```

Comment:
Can we replace or remove the `-n xgboost` argument? It might be clearer to use `conda install -n my_environment` or remove the use of this argument. The use of `xgboost` prompted me to verify that the `-n` specifies the name of an environment instead of the name of the package.

> Target Environment Specification:
>  -n ENVIRONMENT, --name ENVIRONMENT
>                        Name of environment.